File: 周赛/PTA/PTA_GPLT/L1-048.py
Ra,Ca = map(int,input().split())
A = [list(map(int,input().split()))for _ in range(Ra)]
Rb,Cb = map(int,input().split())
B = [list(map(int,input().split()))for _ in range(Rb)]
if Ca != Rb:
    print(f"Error: {Ca} != {Rb}")
else:
    C = [[0 for _ in range(Cb)]for __ in range(Ra)]
    for i in range(Ra):
        for j in range(Cb):
            Multiply = 0
            for k in range(Ca):
                Multiply += A[i][k] * B[k][j];
            C[i][j] = Multiply
    print(f"{Ra} {Cb}")
    for i in range(Ra):
        print(" ".join(map(str,C[i])))

File: 周赛/PTA/PTA_GPLT/L1-048.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    int Ra,Ca,Rb,Cb;
    int i,j,k;
    int Multiply;
    cin >> Ra >> Ca;
    vector<vector<int>>A(Ra,vector<int>(Ca));
    for(i=0;i<Ra;i++){
        for(j=0;j<Ca;j++){
            cin >> A[i][j];
        }
    }
    cin >> Rb >> Cb;
    vector<vector<int>>B(Rb,vector<int>(Cb));
    for(i=0;i<Rb;i++){
        for(j=0;j<Cb;j++){
            cin >> B[i][j];
        }
    }
    if(Ca != Rb){
        cout << "Error: " << Ca << " != " << Rb << endl;
    }
    else{
        vector<vector<int>>C(Ra,vector<int>(Cb));
        for(i=0;i<Ra;i++){
            for(j=0;j<Cb;j++){
                Multiply = 0;
                for(k=0;k<Ca;k++){
                    Multiply += A[i][k] * B[k][j];
                }
                C[i][j] = Multiply;
            }
        }
        cout << Ra << " " << Cb << endl;
        for(i=0;i<Ra;i++){
            for(j=0;j<Cb-1;j++){
                cout << C[i][j] << " ";
            }
            cout << C[i][j] << endl;
        }
    }
}